pkg/opensearch/models.go

```diff
@@ -7,14 +7,15 @@ import (
 
 // Query represents the time series query model of the datasource
 type Query struct {
-	RawQuery   string       `json:"query"`
-	QueryType  string       `json:"queryType"`
-	BucketAggs []*BucketAgg `json:"bucketAggs"`
-	Metrics    []*MetricAgg `json:"metrics"`
-	Alias      string       `json:"alias"`
-	Interval   string
-	RefID      string
-	Format     string
+	RawQuery        string `json:"query"`
+	QueryType       string `json:"queryType"`
+	luceneQueryType string
```

Comment:
This adds `luceneQueryType` which is part of the query object sent to the backend which = 'Traces' for this type of query.